src/index.test.ts

```diff
@@ -1,3 +1,3 @@
 test('basic test', () => {
-  expect(2 + 3 + 6).toBe(12);
+  expect(2 + 3 + 6).toBe(16);
```

Comment:
How does it handle if the tests are inside of a describe block? I think what we'd want is to have each describe section expand into its sub-tests, also.